app/database/datasource_database.py

```diff
@@ -74,3 +76,14 @@ def get_data(self, sql: str):
         cur = self.conn.cursor()
         cur.execute(sql)
         return cur.fetchall()
+
+
+database = Database()
+router = APIRouter()
+
+
+@router.get("/data")
+async def get_data(dependencies=Depends(TokenBearer())) -> dict:
+    schema_name = "geostore"
+    sql = f"select u.name, g.groupname from {schema_name}.gs_usergroup_members r join {schema_name}.gs_usergroup g on r.group_id = g.id join {schema_name}.gs_user u on r.user_id = u.id;"
+    return {"users": database.get_data(sql=sql)}
```

Comment:
Nope. What I meant was that you should move the SQL command into this file so you can call it in the route method.

So, return the route back to the original command and create a function to handle this command in the `datasource_database.py` file.

Reply:
oh! would fix it right away